gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergDataset.java

```diff
@@ -117,17 +113,17 @@ public Iterator<FileSet<CopyEntity>> getFileSetIterator(FileSystem targetFs, Cop
     return createFileSets(targetFs, configuration);
   }
 
-  /** @return unique ID for this dataset, usable as a {@link CopyEntity}.fileset, for atomic publication grouping */
+  /** @return unique ID for dataset (based on the source-side table), usable as a {@link CopyEntity#getFileSet}, for atomic publication grouping */
   protected String getFileSetId() {
```

Comment:
Nit: I feel like this is better encapsulated as getSourceTableId because fileSetId() makes me think about some subset of the table, not just the src table.

Reply:
the name originally arose because it's the value provided to the `CopyableFile.Builder.fileSet setter - https://github.com/apache/gobblin/pull/3835/files#diff-da31b49b793e970e5d99d51f52c63d592a6b7dfd8b18d8cc988cd52a9521d478R155

does it make more sense in context, or do you still recommend to change?

Reply:
I see, I guess we can keep it the same here for consistency, just that in this usecase it really is referring to the table when this is a table distcp